### PR TITLE
Add landing page to exporter

### PR DESCRIPTION
**Pull request by ashleyprimo**
This is a PR per issue #69 to add a landing page to the exporter instead of a 404 error page :smiley: (sorry for the failed previous PR)

### CODE REVIEW

File: ha_cluster_exporter.go [hunk 1]
```diff
@@ -94,6 +94,21 @@ func main() {
...
+			</head>
+			<body>
+				<h1>HACluster Exporter</h1>
+				<p><a href="/metrics">Metrics</a></p>
+				<br />
+				<h2>More information:</h2>
+				<p><a href="https://github.com/ClusterLabs/ha_cluster_exporter">github.com/ClusterLabs/ha_cluster_exporter</a></p>
+			</body>
+			</html>`))
+	})
```
Comment (stefanotorresi) [inline ha_cluster_exporter.go]: could you extract the handler function into a dedicated one? I'd like to keep the main as lean as possible.

File: ha_cluster_exporter.go [hunk 2]
```diff
@@ -94,6 +94,21 @@ func main() {
...
 	}
 
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`<html>
+			<head>
+				<title>HACluster Exporter</title>
+			</head>
+			<body>
+				<h1>HACluster Exporter</h1>
+				<p><a href="/metrics">Metrics</a></p>
```
Comment (stefanotorresi) [inline ha_cluster_exporter.go]: I would make this a relative link, we don't know if this is going to be deployed under a base path or not.

**Comment by MalloZup** [comment]
@ashleyprimo can we have a screenshot attached to see how it looks like?

optional: can you check why is travis failing? thx  sofar!

**Comment by MalloZup** [comment]
https://github.com/ClusterLabs/ha_cluster_exporter/issues/74

**Comment by ashleyprimo** [comment]
@MalloZup 
> @ashleyprimo can we have a screenshot attached to see how it looks like?

You can take a look at a preview image here, [https://ashleyprimo.com/uploads/JEwPXOU3iJDAy1K0raaq.png](https://ashleyprimo.com/uploads/JEwPXOU3iJDAy1K0raaq.png) - It's fairly simple however happy to make changes as needed. :) 

In regards to the Travis CI failure, preliminary checks suggest that it's related to lack of `gofmt -w .` of my code; the diff after doing so is:
```
diff --git a/ha_cluster_exporter.go b/ha_cluster_exporter.go
index fce2e44..5e51291 100644
--- a/ha_cluster_exporter.go
+++ b/ha_cluster_exporter.go
@@ -114,4 +114,4 @@ func main() {
        http.Handle("/metrics", promhttp.Handler())
        log.Infoln("Serving metrics on port", *portNumber)
        log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", *portNumber), nil))
-}
\ No newline at end of file
+}
diff --git a/vendor/github.com/sirupsen/logrus/terminal_check_bsd.go b/vendor/github.com/sirupsen/logrus/terminal_check_bsd.go
index 3c4f43f..ff6ff7b 100644
--- a/vendor/github.com/sirupsen/logrus/terminal_check_bsd.go
+++ b/vendor/github.com/sirupsen/logrus/terminal_check_bsd.go
@@ -10,4 +10,3 @@ func isTerminal(fd int) bool {
        _, err := unix.IoctlGetTermios(fd, ioctlReadTermios)
        return err == nil
 }
-
```

**Comment by MalloZup** [comment]
@ashleyprimo nice thx!
 LGTM we will merge this PR after we release the current master.

good work!

**Pull request merged**